src/Command/ValidateCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Overblog\GraphQLBundle\Command;
+
+use GraphQL\Error\InvariantViolation;
+use GraphQL\Type\Schema;
+use Overblog\GraphQLBundle\Resolver\TypeResolver;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ValidateCommand extends Command
+{
+    private $typeResolver;
+
+    public function __construct(TypeResolver $typeResolver)
+    {
+        parent::__construct();
+        $this->typeResolver = $typeResolver;
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->setName('graphql:validate')
+            ->setDescription('Validate schema')
+        ;
```

Comment:
The bundle allow multiple schema definition so this should also allow it, [see this command](https://github.com/overblog/GraphQLBundle/blob/master/src/Command/GraphQLDumpSchemaCommand.php#L40-L45) for a concrete example.